source/tutorials/conda-tutorial.rst

```diff
@@ -25,14 +25,12 @@ Quickstart
     .. code-block:: bash
 
         conda-build pyinstrument/
```

Comment:
related to https://github.com/cduvallet/q2-perc-norm/pull/4

```suggestion
         conda-build recipe/
```

There are a few other places in the document that reference `pyinstrument`, but I can't comment on those lines in this code review, but I think they can all be updated to `recipe`.

Reply:
Ok! Didn't realize that was the convention.